youtube_dl/YoutubeDL.py

```diff
@@ -848,6 +850,22 @@ def iter_playlistitems(format):
                 random.shuffle(entries)
 
             for i, entry in enumerate(entries, 1):
+                if self.params.get('playlistprompt', False):
+                    if entry['title']:
+                        skipnextvid = False
+                        while True:
+                            answer = compat_input('Do you want to download video %s/%s: "%s"? (y/n) ' %
+                                                  (i, n_entries, entry['title'].encode('ascii', 'ignore')))
```

Comment:
> compat_input can't cope with non-ascii titles

Any Python bug report for that? Maybe we need to implement our ```compat_input``` rather than wrapping existing functions.